<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>游乐园票务系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
   #centralwidget {
    background-image: url(:/images/123.png);
    background-repeat: no-repeat;
    background-position: center 10%;
    background-size: cover;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>80</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>24</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QLabel {
    color: #2c3e50;
    background-color: rgba(255, 255, 255, 0.8);
    padding: 15px;
    border-radius: 10px;
}</string>
      </property>
      <property name="text">
       <string>游乐园售票系统</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_2">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>60</height>
       </size>
      </property>
     </spacer>
    </item>
    <item>
     <widget class="QPushButton" name="queryButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(108, 123, 139, 0.7);
    color: white;
    border: 2px solid rgba(90, 103, 117, 0.9);
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgba(90, 103, 117, 0.9);
    border: 2px solid rgba(74, 85, 104, 0.9);
}
QPushButton:pressed {
    background-color: rgba(74, 85, 104, 0.9);
}</string>
      </property>
      <property name="text">
       <string>查询票价</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="buyTicketButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(108, 123, 139, 0.7);
    color: white;
    border: 2px solid rgba(90, 103, 117, 0.9);
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgba(90, 103, 117, 0.9);
    border: 2px solid rgba(74, 85, 104, 0.9);
}
QPushButton:pressed {
    background-color: rgba(74, 85, 104, 0.9);
}</string>
      </property>
      <property name="text">
       <string>购买门票</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="exitButton">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>50</height>
       </size>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton {
    background-color: rgba(108, 123, 139, 0.7);
    color: white;
    border: 2px solid rgba(90, 103, 117, 0.9);
    border-radius: 10px;
    padding: 10px;
    font-weight: bold;
}
QPushButton:hover {
    background-color: rgba(90, 103, 117, 0.9);
    border: 2px solid rgba(74, 85, 104, 0.9);
}
QPushButton:pressed {
    background-color: rgba(74, 85, 104, 0.9);
}</string>
      </property>
      <property name="text">
       <string>退出系统</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer_3">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>80</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
